pkgs/tools/security/pcsclite/default.nix

```diff
@@ -43,11 +42,9 @@ stdenv.mkDerivation rec {
     "--with-systemdsystemunitdir=${placeholder "bin"}/lib/systemd/system"
   ];
 
-  postConfigure = ''
-    sed -i -re '/^#define *PCSCLITE_HP_DROPDIR */ {
```

Comment:
Note for reviewers: removed because can be adjusted upstream since 2.x, see https://salsa.debian.org/rousseau/PCSC/-/blob/master/ChangeLog#L15